README.md

```diff
@@ -8,6 +8,10 @@ Effortless migration
 
 This repo will contain code and documentation to assist in migrations and upgrades of OpenSearch clusters.
 
+## Setup for commits
+
+Developers must run the "setup.sh" script in order to add the pre-commit hook.
```

Comment:
Out of curiosity: flake8's docs recommend a different strategy for pre-commit hooks https://flake8.pycqa.org/en/latest/user/using-hooks.html. Why did you decide on this approach? (not a problem, just curious)

Reply:
I decided on this since it's the most often used method of adding githooks to a repo in general per my findings, but the strategy you linked is very interesting actually.